code/XMLTester.xml

```diff
@@ -1008,14 +1008,18 @@
 
         <EDP>SIMPLE</EDP>  <!-- string -->
 
-        <leaseDuration>  
+        <leaseDuration>
             <!-- DURATION -->
         </leaseDuration>
 
-        <leaseAnnouncement>  
+        <leaseAnnouncement>
             <!-- DURATION -->
         </leaseAnnouncement>
 
+        <initialAnnouncements>
+            <!-- DURATION -->
+        </initialAnnouncements>
```

Comment:
When we say `<!-- DURATION -->` inside `<initialAnnouncements>`, is it correct? I thought we would have `<count>` and `<period>` instead

Reply:
Isn't correct. Nice catch!